src/rgw/setup/rgw_service

```diff
@@ -37,9 +37,13 @@ while [ $# -gt 0 ];  do
     shift 1
 done
 
+log_dir="$(dirname "${log_file}")"
+motr_log_path="$log_dir/m0_client"
+mkdir -p "$motr_log_path"
+
 function start_rgw {
   machine_id=$(cat /etc/machine-id)
-  exec sudo /usr/bin/radosgw -f --name client.rgw-"$index" -c "$rgw_config_file" --no-mon-config &> "$log_file" &
+  exec sudo M0_TRACE_DIR="$motr_log_path" /usr/bin/radosgw -f --name client.rgw-"$index" -c "$rgw_config_file" --no-mon-config &> "$log_file" &
```

Comment:
By exporting M0_TRACE_DIR using subprocess call doesn't work, it still generates m0trace files in cwd.
but by setting the env while running the command works fine.

> ll /etc/cortx/log/rgw/e884506cadb150157e923ad8e4873ac4/m0_client/
total 16448
-rwx------ 1 root root 16842752 Mar  6 23:10 m0trace.1452814

@cdeshmukh @sachitanands

Reply:
@gargpalak048 check https://github.com/Seagate/cortx-rgw-integration/pull/54/files, this file getting removed. Would need to make these changes based on whats being targeted there

Reply:
@gargpalak048 regarding m0trace log file directory. please check with @yeshpal-jain-seagate once..
there might need some fix in SAL/Motr  layer for this issue

Reply:
@sachitanands setting this env in the command works fine

Reply:
@sachitanands set env M0_TRACE_DIR while starting the service will redirect the log files to the dedicated path

Reply:
Motr was planning to put this M0_TRACE_DIR  as part of config right, any words on that ? 
Env variable setting doesnt give predictable o/p

if no other way right now, we will go ahead by this

Reply:
For a server, there is a sysconfig file available where we can specify configuration parameter, when the server starts, it would read from the configuration file and set those environment. 
libmotr is a library, the user who ever is using can export it as an environment variable.
As of now, there is no plan to read any configuration file to set those parameter, whoever is the consumer of libmotr can modify them in the init scripts/service scripts, that is how is done today in motr ioservice init scripts
@madhavemuri  : are you aware of any requirement

Reply:
sure @yeshpal-jain-seagate, lets discuss this in one dedicated meet, plz schedule one